pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -731,9 +731,16 @@ const storeElasticsearchTemplate = `{{ define "storeElasticsearch" -}}
   port {{.Port}}
   verify_es_version_at_startup false
 {{- if .Target.Secret }}
+  {{- if .Secret }}
+    {{- if and (index .Secret.Data "username") (index .Secret.Data "password") }}
+  user {{ index .Secret.Data "username" }}
+  password {{ index .Secret.Data "password" }}
```

Comment:
This needs to load the password at runtime otherwise its exposed in the configuration. Ref: https://github.com/openshift/cluster-logging-operator/blob/tech-preview/pkg/generators/forwarding/fluentd/templates.go#L634